pandas/io/pytables.py

```diff
@@ -3785,15 +3785,13 @@ def create_axes(
             )
 
         # create according to the new data
-        self.non_index_axes = []
-        self.data_columns = []
+        nia: List = []
```

Comment:
Any chance of adding a subtype here?

Reply:
why the rename? I would imagine `non_index_axes` newcomer friendly than a TLA.

Reply:
> why the rename? I would imagine non_index_axes newcomer friendly than a TLA.

i wanted something not-identical to the self.foo name for grepping purposes, but can try to find a middle ground that is more descriptive

> Any chance of adding a subtype here?

I'll give it the ol' college try.